docs/pages/eas-update/code-signing.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Code Signing
+---
+
+import Head from '~/components/Head'
+
+<Head title="EAS Update Code Signing - Expo Documentation" />
+
+> ⚠️ Expo Updates code signing is in early beta, meaning that it is not yet ready for use in production apps. We may still make breaking changes to the developer-facing portion and the end-user portion as well.
+
+> EAS Update Code Signing is only available to accounts subscribed to the EAS Enterprise plan. [Sign up](https://expo.dev/pricing).
+
+## Introduction
+
+The `expo-updates` library supports end-to-end code signing. Code signing allows developers to cryptographically sign their updates with their own keys. The signatures are then verified on the client before the update is applied, which ensures ISPs, CDNs, cloud providers, and even EAS itself cannot tamper with updates run by apps.
+
+## Code Signing with EAS Update
+
+1. Generate a private key and corresponding code signing certificate for your app:
+
+    ```bash
+    yarn expo-updates codesigning:generate \
+      --key-output-directory keys \
+      --certificate-output-directory certs \
+      --certificate-validity-duration-years 10 \
+      --certificate-common-name "My App"
+    ```
+
+    The generated private key should be kept private and secure.
+
+2. Configure your app's builds to use code signing:
+
+    ```bash
+    yarn expo-updates codesigning:configure \
+      --certificate-input-directory certs \
+      --key-input-directory keys
+    ```
+
+    After this step, a new build (with a new runtime version) should be created. The code signing certificate will be embedded in this new build.
```

Comment:
"should be created" could mean that the command above should create a build for you.
```suggestion
    After this step, create a new build (with a new runtime version). The code signing certificate will be embedded in this new build.
```